mmcls/models/heads/multi_task_head.py

```diff
@@ -83,9 +86,11 @@ def loss(self, feats: Tuple[torch.Tensor],
                     masked_features = masked_features + (feature[mask], )
             else:
                 masked_features = feats[mask]
-            head_loss = head.loss(masked_features, masked_data_samples,
-                                  **kwargs)
-
+            if len(masked_data_samples) > 0:
+                head_loss = head.loss(masked_features, masked_data_samples,
+                                      **kwargs)
+            else:
+                head_loss = {'loss': torch.tensor(0)}
```

Comment:
How do we know that the name of the loss is "loss" ?

```suggestion
                head_loss = {'loss': torch.tensor(0)}
```

Reply:
https://github.com/marouaneamz/mmclassification/blob/31c67ffed406331ab232796b3fdb72f0684b1442/mmcls/models/heads/cls_head.py#L94

Reply:
Normally it should be  head_loss = {task_name_'loss': torch.tensor(0)}